### PR TITLE
Add padding-around-test-blocks

### PR DESCRIPTION
**Pull request by benkimpel**

### CODE REVIEW

**Comment by benkimpel** [comment]
Ready for a look, but there might be some Circle issues. "Waiting for status to be reported". Maybe there were some changes or changes are happening now to Circle config. I'm seeing a workflow that I don't think I saw before.

**Comment by dangreenisrael** [comment]
That was odd. The reported test seemed to change names on the circle side of things. I fixed it in settings

File: docs/rules/padding-around-test-blocks.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Rule Details
+
+This rule enforces a line of padding before _and_ after 1 or more `test`/`it` statements
+
+Note that it doesn't add/enforce a padding line if it's the last statement in its scope
+
+Examples of **incorrect** code for this rule:
+
+```js
```
Comment (dangreenisrael) [inline docs/rules/padding-around-test-blocks.md]: These examples appear to be identical

File: docs/rules/padding-around-test-blocks.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+const thing = 123;
+it('foo', () => {});
+it('bar', () => {});
+```
+
+Examples of **correct** code for this rule:
+
+```js
+const thing = 123;
```
Comment (dangreenisrael) [inline docs/rules/padding-around-test-blocks.md]: Again, these appear to be identical

File: docs/rules/padding-around-test-blocks.md (hunk 1 above)
Comment (benkimpel) [reply docs/rules/padding-around-test-blocks.md]: I did one set with `test` and one with `it`, just to show that both are supported. But lemme know if I'm misunderstanding the comment!

Comment (dangreenisrael) [reply docs/rules/padding-around-test-blocks.md]: ahhhhh. Just me being an idiot

Thanks!

Comment (benkimpel) [reply docs/rules/padding-around-test-blocks.md]: Nah. Your mind just knows it's an alias. You're too deep in the matrix.

**Comment by benkimpel** [comment]
@dangreenisrael sorry for the additional review request... I updated from master and force pushed. That dismissed approval.

**Pull request merged**